pkgs/games/heroic/fhsenv.nix

```
@@ -0,0 +1,98 @@
+{ lib
+, buildFHSUserEnv
+, heroic-unwrapped
+}:
+
+buildFHSUserEnv {
+  name = "heroic";
+
+  runScript = "heroic";
+
+  targetPkgs = pkgs: with pkgs; [
+    heroic-unwrapped
+
+    gnugrep
+    gnused
+    gnutar
+    zstd
+    opencl-headers
+    p7zip
+    perl
+    psmisc
+    which
+    xorg.xrandr
+  ];
+
+  multiPkgs = let
+    xorgDeps = pkgs: with pkgs.xorg; [
+      libX11 libXrender libXrandr libxcb libXmu libpthreadstubs libXext libXdmcp
+      libXxf86vm libXinerama libSM libXv libXaw libXi libXcursor libXcomposite
+    ];
+  in pkgs: with pkgs; [
+    alsa-lib
+    bash
+    cairo
+    coreutils
+    cups
+    dbus
+    freetype
+    fribidi
+    giflib
+    glib
+    gnutls
+    gtk3
+    lcms2
+    libevdev
+    libGLU
+    libglvnd
+    libjpeg
+    libkrb5
+    libmpeg2
+    libogg
+    libopus
+    libpng
+    libpulseaudio
+    libselinux
+    libsndfile
+    libsndfile
+    libtheora
+    libtiff
+    libusb1
+    libv4l
+    libva
+    libvorbis
+    libxkbcommon
+    libxml2
+    mpg123
+    ocl-icd
+    openldap
+    samba4
+    sane-backends
+    SDL2
+    udev
+    udev
+    unixODBC
+    util-linux
+    vulkan-loader
+    wayland
+    zlib
+  ] ++ xorgDeps pkgs;
+
+  meta = {
+    inherit (heroic-unwrapped.meta)
+      homepage
+      description
+      platforms
+      license
+      maintainers
+      broken;
+
+    mainProgram = "heroic";
+  };
```

```suggestion
  meta = heroic-unwrapped.meta // {
    mainProgram = "heroic";
  };
```
also meta should be last.